fbw-a380x/src/systems/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -957,6 +957,59 @@ class V1Offtape extends DisplayComponent<{ bus: EventBus }> {
   }
 }
 
+class ArsBar extends DisplayComponent<{ bus: ArincEventBus }> {
+  private arsPath = Subject.create<string>('');
+
+  private arsVisiblity = Subject.create<string>('hidden');
+
+  private airSpeed = new Arinc429Word(0);
+
+  private conf1FSelected = false;
+
+  private readonly ARS1FSpeed = 212;
+
+  private readonly Conf1FVFE = 222;
+
+  private readonly offset = ((this.ARS1FSpeed - this.Conf1FVFE) * DistanceSpacing) / ValueSpacing;
+
+  private setArspath() {
+    if (this.conf1FSelected) {
+      this.arsVisiblity.set('visible');
+      const arsPos = ((this.airSpeed.value - this.ARS1FSpeed) * DistanceSpacing) / ValueSpacing + 80.818;
+      this.arsPath.set(`m19.031 ${arsPos}h 1.9748v${this.offset}`);
```

Comment:
It would be much more efficient to transform this rather than changing the path constantly, but I think we really have that problem all over the PFD so I guess it's okay for now.